bcgov/onroutebc
# Orv2 763 1

# Description

This PR integrated soft delete applications by updating applications status to Cancelled.  

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?
Test steps:

1. Navigate to Permits tab.
2. Click "Start Application" to create a new application.
3. Fill out required fields and save application.
4. Click "Permits" tab to display all the applications in progress. 
5. Create multiple applications by following step 1-3.
6. Select/deselect the rows for each application from the application list view.
7. Click the trash icon on each row, or on the search bar.
8. When clicking on the trash icon of each row, expect that the application of that specific row will be deleted.
9. When clicking on the top of the search bar, expect all the selected applications will be deleted.


## Checklist






## Further comments




---

Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Backend - Vehicles](https://onroutebc-452-backend-vehicles.apps.silver.devops.gov.bc.ca/) available
[Frontend](https://onroutebc-452-frontend.apps.silver.devops.gov.bc.ca/) available

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/onroutebc/actions/workflows/merge-main.yml)
